docs/book/src/reference/versions.md

```diff
@@ -85,6 +85,7 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.27  | ✓ >= v1.4.2          | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.28  |                      | ✓ >= v1.5.1       | ✓                 | ✓                 |
 | Kubernetes v1.29  |                      |                   | ✓ >= v1.6.1       | ✓                 |
+| Kubernetes v1.30  |                      |                   |                   | ✓ >= v1.7.1       |
```

Comment:
This needs to be duplicated in the other two tables below

At some point we should probably figure out a way to deduplicate. It doesn't make much sense